docs/explanation/taxon-constraints-explainer.md

```diff
@@ -2,67 +2,307 @@
+
+- `'whisker' SubClassOf (in_taxon some (not 'Hominidae'))`
+
+Now both HermiT and ELK can find 'whisker muscle in human' to be unsatisfiable. This is the explanation from ELK:
+
+- `'whisker muscle in human' EquivalentTo ('whisker muscle' and (in_taxon some 'Homo sapiens'))`
+- `'Homo sapiens' SubClassOf 'Hominidae'`
+- `'whisker muscle' SubClassOf (part_of some 'whisker')`
+- `'whisker' SubClassOf (in_taxon some ('not 'Hominidae'))`
+- `part_of o in_taxon SubPropertyOf in_taxon`
+- `(in_taxon some 'Hominidae') DisjointWith (in_taxon some (not 'Hominidae'))`
+
+**In summary, the follow constructs are all needed for QC using taxon restrictions:**
+- Relation Ontology
+  - `in_taxon` property chains for relations which should propagate `in_taxon` inferences
+- NCBI Taxonomy Ontology
+  - `X DisjointWith Y` for all sibling taxa `X` and `Y`
+  - `(in_taxon some X) DisjointWith (in_taxon some Y)` for all sibling taxa `X` and `Y`
+  - `(in_taxon some X) DisjointWith (in_taxon some (not X))` for every taxon `X`
+- Each ALL-IN taxon restriction `C in_taxon X`
+  - `C SubClassOf (in_taxon some X)`
+- Each NOT-IN taxon restriction `C never_in_taxon X`
+  - `C SubClassOf (not (in_taxon some X))`
+  - `C SubClassOf (in_taxon some (not X))`
+
+## Exploring taxon restrictions in Protégé
+
+Using the DL Query panel and a running reasoner, it is straightforward to check whether a particular taxon restriction holds for a term (such as when someone has requested one be added to your ontology). Given some term of interest, e.g., 'whisker', submit a DL Query such as `'whisker' and (in_taxon some Mammalia)`. Check the query results:
+- If `Equivalent classes` includes `owl:Nothing`, then a never_in_taxon is implied for that taxon.
+- If `Equivalent classes` includes the term of interest itself (and not `owl:Nothing`), then an in_taxon is implied for that taxon.
+- If `Superclasses` includes the term of interest (and the query isn't equivalent to `owl:Nothing`), then there is no particular taxon restriction involving that taxon.
+
+### OBO taxon constraints plugin for Protégé
+
+To quickly see exactly which taxon restrictions are in effect for a selected term, install the [OBO taxon constraints plugin for Protégé](https://github.com/geneontology/protege-taxon-constraints). Once you have the plugin installed, you can add it to your Protégé window by going to the menu `Window > Views > OBO views > Taxon constraints`, and then clicking the location to place the panel. The plugin will show the taxon constraints in effect for the selected OWL class. When a reasoner is running, any inferred taxon constraints will be shown along with directly asserted ones. The plugin executes many reasoner queries behind the scenes, so there may be a delay before the user interface is updated.
+
+<img width="1648" alt="image" src="https://user-images.githubusercontent.com/210210/211460355-262c637e-dfd7-4dc3-b301-04d80ee47ddf.png">
+
+## How to add taxon restrictions:
```

Comment:
Could I suggest we move this up to before `## Using taxon restrictions for Quality Control` - I understand the logic of doing it this way, but I'm thinking more on the side of an ontology editor which might get intimidated by the new additions. Would like to funnel them into the how-to before that happens and all.